python/feathub/processors/spark/dataframe_builder/time_utils.py

```diff
@@ -46,7 +46,7 @@ def append_unix_time_attribute_column(
         return df.withColumn(
             EVENT_TIME_ATTRIBUTE_NAME,
             functions.expr(
-                f"unix_millis(to_timestamp("
-                f"`{timestamp_field}`,'{java_datetime_format}'))"
+                f"unix_millis(from_utc_timestamp(to_timestamp("
```

Comment:
Can you check if this method can handle the case where the input contains information about the timezone? If not, we may need to add a TODO for that case, similar to python/feathub/processors/flink/table_builder/source_sink_utils_common.py#64